server/src/main/webapp/WEB-INF/rails/webpack/models/shared/job_state.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 ThoughtWorks, Inc.
```

Comment:
One miniscule thing I noticed afterwards (fixed myself, so no worries), assuming you are using IDEA, it seems the IDE hasn't automagically picked up the [new Copyright profile](https://github.com/gocd/gocd/commit/f7161e8a6a2af7ba330c67a440d754cf8301fc85) with tweaked capitalization. Do you know if I did something wrong here, or you perhaps have some IDE-level settings your side?

Reply:
I did not restart my intellij since a few weeks. So I suppose it could have been stale state in my IDE. I'll fix it the next commit.

Reply:
No worries - I already fixed it in https://github.com/gocd/gocd/pull/10716/commits/bc5b2d513857582e569d30a29ead29882e982ac1 in case it was my bad :-)